cpp/src/io/parquet/parquet_gpu.hpp

```diff
@@ -412,6 +428,15 @@ struct ColumnChunkDesc {
 
   int32_t src_col_index{};   // my input column index
   int32_t src_col_schema{};  // my schema index in the file
+
+  float list_bytes_per_row_est{};  // for LIST columns, an estimate on number of bytes per row
+};
+
+/**
+ * @brief A utility structure for use in decoding page headers.
+ */
+struct chunk_page_info {
+  PageInfo* pages;
```

Comment:
This is just a type aliasing right?

Reply:
See https://github.com/rapidsai/cudf/pull/14360#discussion_r1459833187